### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by brytebee**
## Styling 🎨🎨
In this PR, I have achieved the following:
- added basic styles to the web app
- installed and used material icons
- installed and used font-awesome
-  installed and used google fonts

### CODE REVIEW

File: src/components/App.css [hunk 1]
```diff
@@ -3,19 +3,36 @@
   margin: 0;
```
Comment (D4nielJ) [inline src/components/App.css:3]: - [x] Let's try to make a couple of subtle changes to make it look a lot more like the original design.

1. Let's add a white bg to the Navbar and a border-bottom gray.
2. Let's change the weight the fonts of the fonts on the Navbar. Title is Montserrat-Bold, you instead applied Montserrat-Black
3. Let's increase the padding vertical a little. We can also center the items making use of Flex.
4. You can also increase a little bit the horizontal padding of the content (cards and form).

Yours:
image

Template:
image

If you have followed mi indications until here, you should end with a more accurate design than the one provided.

Yours:
image


After a few minor twerks:
image

**Review by D4nielJ — CHANGES_REQUESTED** [review]
## Hello @brytebee, 

You're doing great! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :fire: :rocket: 
- Styling does not affect the correct behavior of the app
- You use the correct colors, fonts, and components!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Happy coding and stay awesome! 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can always find me on slack!

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/App.css (hunk 1 above)
Comment (brytebee) [reply src/components/App.css:3]: Done! Thanks for pointing them out, I have fixed them.

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @brytebee 

Great work on making the project requirements 👏 
You've done well implementing some of the project requirements, but there are still some which aren't addressed yet, please work on it and ask for another review.


 ## Required Changes ♻️
 
- As requested by the previous reviewer, please try to match with the design provided. The header, progress, list of the book added, add button, and an input field.

hello-3


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**